contracts/eth/nearprover/utils/eth-ledger-signer.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2019 Richard Moore
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction,
+// including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions
+// of the Software.
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+// WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+// OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// Note: this code copied and edited from 'ethers' project due to missing EIP1559 support.
+
+const ethers = require('ethers');
+const hwAppEth = require('@ledgerhq/hw-app-eth').default;
+const hid = require('@ledgerhq/hw-transport-node-hid').default;
+const defaultPath = "m/44'/60'/0'/0/0";
+function waiter(duration) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, duration);
+    });
+}
+class EthLedgerSigner extends ethers.Signer {
+    constructor(provider, path) {
+        super();
+        if (!path) {
+            path = defaultPath;
+            console.log(defaultPath)
+        }
+        ethers.utils.defineReadOnly(this, 'path', path);
+        ethers.utils.defineReadOnly(this, 'provider', provider || null);
+        ethers.utils.defineReadOnly(this, '_eth', hid.create().then((transport) => {
+            const eth = new hwAppEth(transport);
+            return eth.getAppConfiguration().then((_config) => {
+                return eth;
+            }, (error) => {
+                return Promise.reject(error);
+            });
+        }, (error) => {
+            return Promise.reject(error);
+        }));
+    }
+    _retry(callback, timeout) {
```

Comment:
Add an empty line here and in other places in the file. Even if this formatting was used in the original repo, it's worth fixing.

Reply:
fixed at e5f44756a4da2b1b886ed1e94421532ed5fa1d42